### PR TITLE
Add Fether FAQ

### PR DESCRIPTION
**Pull request by Tbaut**
Closes https://github.com/paritytech/wiki/issues/204

### CODE REVIEW

File: Fether-FAQ.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Fether - Frequently Asked Questions
+libs:
+  faq: true
+---
+
+### Why is the sync so slow?
+
+Parity Fether accesses the blockchain in a decentralised manner by using Parity Ethereum light client.
+While we expect to be able to sync with the top of the blockchain in a manner of seconds in the near future, some underlying optimisations need to be made.
```
Comment (amaurymartiny) [inline Fether-FAQ.md]: in a matter of seconds?

Comment (Tbaut) [reply Fether-FAQ.md]: with a recent enough hardcoded headers we're talking about less than a min. And this lasts for a week or so :)

Comment (amaurymartiny) [reply Fether-FAQ.md]: Ah no, I meant manner -> matter

Comment (Tbaut) [reply Fether-FAQ.md]: haha nice, totally missed it, so did Grammarly..

**Pull request merged**